snippets/csharp/VS_Snippets_VBCSharp/csrefKeywordsTypes/CS/keywordsTypes.cs

```diff
@@ -87,64 +87,49 @@ static void Main()
     interface IPoint
     {
        // Property signatures:
-       int x
+       int X
```

Comment:
Isn't this an instance auto-implemented property? I thought this wasn't allowed.

Reply:
In an interface definition, this defines the contract, it doesn't define an implementation. 

The sample is correct. I'll update the article text to make that more clear. Good catch!